.github/workflows/code-coverage.yml

```diff
@@ -6,5 +6,5 @@ jobs:
   workflow-call:
     uses: grafana/code-coverage/.github/workflows/code-coverage.yml@v0.1.10
     with:
-      frontend-path-regexp: src
+      frontend-path-regexp: 
```

Comment:
Code coverage temporarily disabled. It doesn't currently get run on main, and getting code coverage to successfully run would require further changes.